models/document-generation.ts

```diff
@@ -559,6 +585,50 @@ const incrementalFundingPlan = {
   },
   additionalProperties: false,
 };
+export const evalPlan = {
+  type: "object",
+  properties: {
+    taskOrderTitle: { type: "string" },
+    sourceSelection: {
+      type: "string",
+      enum: [
+        SourceSelection.NO_TECH_PROPOSAL,
+        SourceSelection.TECH_PROPOSAL,
+        SourceSelection.SET_LUMP_SUM,
+        SourceSelection.EQUAL_SET_LUMP_SUM,
+      ],
+    },
+    method: {
+      type: "string",
+      enum: [EvalPlanMethod.BEST_USE, EvalPlanMethod.LOWEST_RISK, EvalPlanMethod.BVTO, EvalPlanMethod.LPTA, null],
```

Comment:
Because of the addition of `null` the `type: "string"` will cause the request to fail validation if the method is `null`.

```suggestion
      enum: [EvalPlanMethod.BEST_USE, EvalPlanMethod.LOWEST_RISK, EvalPlanMethod.BVTO, EvalPlanMethod.LPTA, null],
```